sdk/identity/ci.yml

```diff
@@ -73,12 +73,139 @@ extends:
     LiveTestStages:
       - template: /eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml
         parameters:
+          PreDeploySteps:
+            - task: AzureCLI@2
+              displayName: 'Resolve paths and run Maven builds'
+              inputs:
+                azureSubscription: azure-sdk-tests-public
```

Comment:
If you hard-code this in the prestep it will block this from working in other subscriptions/clouds.